cartographer_ros_msgs/msg/SubmapEntry.msg

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-uint32 submap_version
+int32 submap_version
```

Comment:
Hi, thanks on the response! 

The changes in this pull request have already been organized in separate, self-standing commits (for example, you can find the answer to this question in the description of the commit "Change SubmapEntry submap_version field to be int32"). I imagined that you would go through each of them, in the order they were commited, if you wanted to tackle them one-by-one (please also have a look at the commit descriptions). If it's easier for you, I can still make separate pull requests; just let me know.

For now, I will go address the style nits now and push them as a new commit.